modules/collection/programs/beets.nix

```
@@ -0,0 +1,77 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}: let
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkOption mkEnableOption mkPackageOption;
+
+  yaml = pkgs.formats.yaml {};
+
+  cfg = config.rum.programs.beets;
+in {
+  options.rum.programs.beets = {
+    enable = mkEnableOption "beets";
+
+    package = mkPackageOption pkgs "beets" {
+      extraDescription = ''
+        To get plugins to work, you will need to override the beets derivation
+        with the plugins you want:
+
+        ```nix
+        package = pkgs.beets.override {
+            pluginOverrides = {
+                fish.enable = true;
+                convert.enable = true;
+            };
+        };
+        ```
+
+        Consult the [beets derivation] for a list of available plugins.
+
+        [beets derivation]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/audio/beets/builtin-plugins.nix
+      '';
+    };
```

```suggestion
        Consult the [beets derivation] for a list of available plugins.

        [beets derivation]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/audio/beets/builtin-plugins.nix
      '';
      example = ''
      pkgs.beets.override {
        pluginOverrides = {
          fish.enable = true;
          convert.enable = true;
        };
        ''
        };
    };
```

Please add this to `example` instead